rpc/documentation/api.md

```diff
@@ -376,6 +384,45 @@ should begin at.
 - `FailedPrecondition`: The wallet or consensus RPC server has not been opened.
 
 **Stability:** Unstable
+___
+
+#### `FetchMissingCFilters`
+
+The `FetchMissingCFilters` method fetches any missing committed filters from
+the main chain.
+
+**Request:** `FetchMissingCFiltersRequest`
+
+**Response:** `FetchMissingCFiltersResponse`
+
+**Expected errors:**
+
+- `FailedPrecondition`: The wallet or consensus RPC server has not been opened.
+
+**Stability:** Unstable
+___
+
+#### `RescanPoint`
+
+The `RescanPoint` method returns the current RescanPoint hash which allows other
```

Comment:
self-referential documentation isn't particularly useful.  This should describe what the rescan point is, not that it returns it.

For example (taken from the Wallet.RescanPoint godoc comment):

>  RescanPoint returns the block hash at which a rescan should begin (inclusive), or nil when no rescan is necessary.

Replace nil with null since this is protobuf not go.